fhirzeug/generators/python_pydantic/static_files/tests/test_extension.py

```diff
@@ -81,3 +96,160 @@ def test_extension_in_a_domain_resource() -> None:
     data["extension"][0]["extension"][1]["extension"] = [{"url": "test/example", "valueInteger": 45}]  # type: ignore
     with pytest.raises(pydantic.ValidationError):
         r4.from_dict(data)
+
+
+def test_primitive_extension_object(
+    human_extension: r4.Extension, integer_extension: r4.Extension
+) -> None:
+    """Test PrimitiveExtension object."""
+    r4.PrimitiveExtension()
+    r4.PrimitiveExtension(id=ID)
+    r4.PrimitiveExtension(extension=[human_extension, integer_extension])
+    r4.PrimitiveExtension(id=ID, extension=[human_extension])
+
+    with pytest.raises(pydantic.ValidationError):
+        # Extension must be a list
+        r4.PrimitiveExtension(extension=human_extension)
+
+
+def test_primitive_extension_basic_usage(primitive_extension: r4.PrimitiveExtension):
+    """Test usage of a PrimitiveExtension."""
+    r4.Patient()
+    r4.Patient(gender="unknown")
+    r4.Patient(gender__extension=primitive_extension)
+    r4.Patient(gender="unknown", gender__extension=primitive_extension)
+
+
+@pytest.mark.parametrize("null_value", [None, "", {}, []])
```

Comment:
I'd rather call it `empty_value` or `falsy_value`.

Reply:
Changed to `empty_value`